deploy/kubernetes/overlays/alpha/rbac_add_resizer.yaml

```diff
@@ -0,0 +1,35 @@
+# Resizer must be able to work with PVCs, PVs, SCs.
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: external-resizer-role
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims/status"]
+    verbs: ["update", "patch"]
+  - apiGroups: ["storage.k8s.io"]
```

Comment:
I think storageclass is not needed anymore.
https://github.com/kubernetes-csi/external-resizer/blob/master/deploy/kubernetes/rbac.yaml